File: src/harkerrobolib/commands/ThrowExceptionCommand.java
package harkerrobolib.commands;

public class ThrowExceptionCommand extends CallMethodCommand {

	public ThrowExceptionCommand(RuntimeException e) {
		super(() -> throwException(e));
	}
	
	public ThrowExceptionCommand (String s) {
		this (new RuntimeException(s));
	}
	public static void throwException (RuntimeException e){
		throw e;
	}

}

File: src/harkerrobolib/commands/ThrowExceptionCommand.kt
package harkerrobolib.commands

import java.io.File
import java.security.InvalidParameterException

import edu.wpi.first.wpilibj.command.Command

class ThrowExceptionCommand(e: RuntimeException) : CallMethodCommand({ throwException(e) }) {

    constructor(s: String) : this(RuntimeException(s)) {}

    companion object {
        fun throwException(e: RuntimeException) {
            throw e
        }
    }

}
